amida-tech/saraswati-cql-execution
# Adding External CIS CQL

# Related Tickets



- [SAR-229](https://amida.atlassian.net/browse/SAR-229)
- [SAR-241](https://amida.atlassian.net/browse/SAR-241)

# Other Repos' PR(s) Intended to Work With This PR

- N/A

# How Things Worked (or Didn't) Before This PR

- There was no external CIS CQL to reference and execute

# How Things Work Now (And How to Test)
- Now there is external an CIS CQL to reference and execute

### Test
- Run `yarn start` in one terminal
- Run `node receiver.js`  in another terminal
- Modify the `external-childhood-immunization-status-patients.json` file and save it. 
- Go to the terminal and you should see the results
